Content.Shared/Maps/TurfHelpers.cs

```diff
@@ -225,5 +225,24 @@ private static Box2 GetWorldTileBox(TileRef turf)
 
             return tileBox;
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="tileRef"></param>
+        /// <param name="entity"></param>
+        /// <param name="coords"></param>
+        /// <returns></returns>
+        public static EntityCoordinates? IsTileClearAndInRange(IMapManager mapManager, IEntity entity, EntityCoordinates coords)
```

Comment:
I would expect a method with this name to return a boolean, not nullable coordinates. What's the purpose of this method? Maybe we can find a better name.